### PR TITLE
git 2.13.1

### PR DESCRIPTION
**Pull request by leonklingele**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/git.rb [hunk 1]
```diff
@@ -37,12 +37,12 @@ class Git < Formula
   end
 
   resource "html" do
-    url "https://www.kernel.org/pub/software/scm/git/git-htmldocs-2.13.0.tar.xz"
+    url "https://www.kernel.org/pub/software/scm/git/git-htmldocs-2.13.1.tar.xz"
     sha256 "d5ddfb8eedd5a53c0e46e183ed1513dbdd4f5d5e7ef6624040ba0b7381221e3a"
```
Comment (ilovezfs) [inline Formula/git.rb]: checksum needs to be updated

File: Formula/git.rb [hunk 2]
```diff
@@ -37,12 +37,12 @@ class Git < Formula
...
   end
 
   resource "man" do
-    url "https://www.kernel.org/pub/software/scm/git/git-manpages-2.13.0.tar.xz"
+    url "https://www.kernel.org/pub/software/scm/git/git-manpages-2.13.1.tar.xz"
     sha256 "8414f9c62e2b099cd0ea2ca22c55fb6538b13fc7a3ed1508fb9881aae42a97c9"
```
Comment (ilovezfs) [inline Formula/git.rb]: checksum needs to be updated

**Comment by leonklingele** [comment]
@ilovezfs yup, sorry! Updated.

**Comment by ilovezfs** [comment]
@leonklingele no problem at all. Thanks for the PR. Shipped!

**Pull request closed without merging**